portality/bll/services/article.py

```diff
@@ -141,6 +141,24 @@ def _prepare_update_publisher(self, article, duplicate, merge_duplicate, account
                 raise exceptions.DuplicateArticleException()
         return is_update
 
+    # here we should have the final point of validation for all incoming articles
+    def _sanity_check(self, article):
```

Comment:
There's already a method in the `create_article` function called `is_acceptable` which does some validation on the object, so I'd say this logic belongs there instead of a new function.

Also, now we're changing the capabilities of the bll, we need to update the bll test, which I can help you with.